kornia/testing/__init__.py

```diff
@@ -260,31 +272,27 @@ def assert_close(
         **kwargs: Any,
     ) -> None:
         if rtol is None and atol is None:
-            with contextlib.suppress(Exception):
-                rtol, atol = _get_default_tolerance(actual, expected)
-
-        return _assert_close(actual, expected, rtol=rtol, atol=atol, check_stride=False, equal_nan=True, **kwargs)
+            # `torch.testing.assert_close` used different default tolerances than `torch.testing.assert_allclose`.
+            # TODO: remove this special handling as soon as https://github.com/kornia/kornia/issues/1134 is resolved.
+            #  Basically, this whole wrapper function can be removed and `torch.testing.assert_close` can be used
+            #  directly.
+            rtol, atol = _default_tolerances(actual, expected)
+
+        return _assert_close(
+            actual,
+            expected,
+            rtol=rtol,
+            atol=atol,
+            # this is the default value for torch>=1.10, but not for torch==1.9
+            # TODO: remove this if kornia relies on torch>=1.10
+            check_stride=False,
+            equal_nan=True,
```

Comment:
This is really dangerous as default, since it can easily hide bugs. You should check if setting this to `False` (or removing it since that is the default) fails tests. Similar to #1134 these tests should be manually inspected and if there is no bug, `equal_nan=True` should only be set there.